support/cas-server-support-discovery-profile/build.gradle

```diff
@@ -6,6 +6,7 @@ dependencies {
     implementation project(":core:cas-server-core-web-api")
     implementation project(":core:cas-server-core-authentication-attributes")
     implementation project(":core:cas-server-core-configuration-api")
+    implementation project(":core:cas-server-core-rest")
```

Comment:
Can't do this. Enabling discovery should not have to auto-enable REST. Why is this needed? and if it is, could we break the rest module down to an API and config module?